src/Router.js

```diff
@@ -15,7 +16,7 @@ const Router = () => {
         <Route path="/" element={<Login />} />
         <Route path="/cart" element={<Cart />} />
         <Route path="/main" element={<Main />} />
-        <Route path="/detail" element={<Detail />} />
+        <Route path="/ProductDetail/:id" element={<ProductDetail />} />
```

Comment:
```suggestion
        <Route path="/productDetail/:id" element={<ProductDetail />} />
```
경로는 다른 경로와 컨벤션을 맞춰서 소문자로 시작해주세요!